app/views/guides/_form.html.erb

```diff
@@ -4,7 +4,7 @@
       <legend>Meta</legend>
       <div class='form-group'>
         <%= f.label :slug %>
-        <%= f.text_field :slug, class: 'input-md-12 form-control' %>
+        <%= f.text_field :slug, class: 'input-md-12 form-control', value: guide.slug || "/service-manual/" %>
```

Comment:
Should we write a feature test to make sure the slug field is pre-filled when creating new guides?

I think I'd also prefer setting this value in the controller `new` action, the same way we set an edition for a new guide https://github.com/alphagov/service-manual-publisher/blob/616661944759aab8563bca2c3ede370c1a732e39/app/controllers/guides_controller.rb#L7